wizeline/wize-q-remix
# feat(migration): add questions commented by user controller with integration tests

#### What does this PR do?
Adds questions commented controller and the necessary tests. 

#### Where should the reviewer start?
app/controllers/profile/questionsCommented.js
tests/integration/controllers/profile/questionCommented.test.js

#### How should this be manually tested?
1. Run the command 'npm run test:integration' in the console and make sure all the test are passed.


#### What are the relevant tickets?
Closes #98 

#### Screenshots
:camera:
![Screen Shot 2022-11-28 at 1 11 51 p m](https://user-images.githubusercontent.com/45604471/204361181-b2d17e56-1382-4323-aeb1-c56bb9ca069d.png)
